### PR TITLE
Add docs header to generated readme.md files

### PR DESCRIPTION
**Pull request by daschult**

### CODE REVIEW

**Comment by olydis** [comment]
# 🤖 AutoRest automatic publish job 🤖
~~~ Haskell
> will publish once PR gets merged
~~~

File: test/options/generatereadmemd-azure-true/Expected/AcceptanceTests/ParameterFlattening/README.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+---
+uid: azure-arm-parameterflattening
+summary: *content
+---
+
```
Comment (dend) [inline test/options/generatereadmemd-azure-true/Expected/AcceptanceTests/ParameterFlattening/README.md]: @daschult can we make sure that there is an empty line after summary, and no empty line after the second `---`?

Comment (daschult) [reply test/options/generatereadmemd-azure-true/Expected/AcceptanceTests/ParameterFlattening/README.md]: Sure. Just out of curiosity, why does the whitespace matter? Is it style/preference, or does your code require it?

Comment (dend) [reply test/options/generatereadmemd-azure-true/Expected/AcceptanceTests/ParameterFlattening/README.md]: @daschult I've noticed that without the whitespace sometimes the system fails to pick up the package information.

Comment (daschult) [reply test/options/generatereadmemd-azure-true/Expected/AcceptanceTests/ParameterFlattening/README.md]: Okay, done. Please take another look.

**Comment by dend** [comment]
LGTM, thanks @daschult

**Pull request merged**